sdk/core/core-tracing/test/interfaces.spec.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
duplicated header

Reply:
thank you! I thought I cleaned these 🤔 woops